test/simulation/simulation_reputer.go

```diff
@@ -0,0 +1,41 @@
+package simulation
+
+import (
+	testCommon "github.com/allora-network/allora-chain/test/common"
+	"time"
+)
+
+const EpochLength = 5
```

Comment:
Now minimum EpochLength = 60 on chain, so this fails to create a topic.
Ofc this can be reduced only if the parameter that sets the minimum epochLength also has to be explicitly reduced - so this can be added (also to stress and integration tests btw) , a check where if the EpochLength received is under the minimum, reset the minimum to that value, just to accomodate tests.

Also, this and other settings should be parameterised like in the stress tests. Here topic.EpochLength is defined in the simulation_reputer file which is not directly related.

Reply:
Thanks for your review. And the reason I didn't parameterize it is because the goal of this simulation is to generate a graph to verify that the loss calculation works as we researched. Ofc, if all settings are parameterized, it can be very helpful to generate several graphs to check the logic.